client/src/components/ResponsiveNav.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class='navbar is-spaced has-shadow'>
+    <div class='navbar-brand'>
+      <router-link class='navbar-item' to='/'>
+        <img src='../assets/lo-logo.png' class='logo' />
+      </router-link>
+      <a role='button' class='navbar-burger' aria-label='menu' aria-expanded='false'>
+        <span aria-hidden='true'></span>
+        <span aria-hidden='true'></span>
+        <span aria-hidden='true'></span>
+      </a>
+    </div>
+    <div class='navbar-menu'>
+      <div class='navbar-end'>
+        <div v-for='route in routes' class='navbar-item nav-link' :key='route[0]'>
+          <router-link :to='route[1]'>{{ route[0] }} </router-link>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { Titles } from '../assets/messages/common';
+import * as router from '../router';
+
+export default defineComponent({
+  name: 'ResponsiveNav',
+  data() {
+    const routes: [string, string][] = [
+      [Titles.HOME_TITLE, router.HOME_ROUTE],
+      [Titles.LEAD_STATUS_TITLE, router.SCORECARD_BASE],
+      [Titles.MAP_TITLE, router.MAP_ROUTE_BASE],
+      [Titles.ABOUT_TITLE, router.ABOUT_ROUTE],
+      [Titles.RESOURCES_TITLE, router.RESOURCES_ROUTE],
+    ];
+    return {
+      routes,
+    };
+  },
+  mounted() {
+    // Expand the menu via the burger button
```

Comment:
so this changes all navbars, not just mobile version?

Reply:
Short answer: yes

Longer answer: there is only one navbar. It just looks different in mobile vs desktop (desktop shows the links while mobile shows an expandable menu via a burger button). Bulma doesn't include any JS so we have to write our own to make the button work here.

Reply:
SG! Just wanted to make sure links still showed on desktop.